src/components/AddBeer.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Header from './Header';
+
+export class AddBeer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      tagline: '',
+      description: '',
+      first_brewed: '',
+      brewers_tips: '',
+      attenuation_level: 0,
+      contributed_by: '',
+    };
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    axios
```

Comment:
Podes generar una instancia de axios en otro archivo para reutilizar por ejemplo

const api = axios.create({
   baseURL: 'https://ih-beers-api2.herokuapp.com',
  headers: {... if you need headers}
})

api.post("/beers/new", ...)

Reply:
Listo, implementados todos los cambios que me has comentado en el repo. Gracias, queda mucho más limpio :)